addons/web_editor/static/lib/odoo-editor/src/commands/deleteForward.js

```diff
@@ -85,6 +85,13 @@ HTMLElement.prototype.oDeleteForward = function (offset) {
         restore();
         HTMLElement.prototype.oDeleteForward.call(grandparent, parentIndex);
         return;
+    } else if (
+        firstLeafNode &&
+        firstLeafNode.nodeType === Node.TEXT_NODE &&
+        firstLeafNode.textContent === '\ufeff'
+    ) {
+        firstLeafNode.oDeleteForward(1);
```

Comment:
It's hard to get rid of orphaned FEFF during edition and we assume they're never legit.

We can get orphaned FEFF for instance in the following case: `//ZWNBSP//[]<a>//ZWNBSP//abc//ZWNBSP//</a>//ZWNBSP//`(\*) + insertChar('A') = `//ZWNBSP//A[]//ZWNBSP//<a>//ZWNBSP//abc//ZWNBSP//</a>//ZWNBSP//`.

We remove these orphaned FEFF whenever we can in `sanitize` but we don't want to impact the selection so that can get tricky. So as not to take unnecessary risks, we only remove them if their closest block isn't in the selection.

(\*) I can put the cursor there by clicking if the link has a big enough margin - same for inside with a padding

Reply:
If you replace `insertChar('A')` with `insertParagraphBreak()` in my example you're gonna have a problem then. The whole point of using FEFF is that we're considering them _never_ to be legit, so we _remove all of them_ on save.

Reply:
Alright.